packages/ui/src/components/pagination/acvPagination.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { describe, expect, it } from 'vitest';
+import { mount } from '@vue/test-utils';
+import { axe } from 'vitest-axe';
+import AcvPagination from './AcvPagination.vue';
+import type { AcvPaginationProps } from './acvPagination.ts';
+
+describe('test AcvPagination component', () => {
+  it('pass accessibility tests', async () => {
+    const wrapper = mount(AcvPagination, {
+      props: {
+        total: 100,
+        limit: 10,
+        modelValue: 1,
+      } as AcvPaginationProps,
+    });
+
+    expect(await axe(wrapper.element)).toHaveNoViolations();
+  });
+
+  it('default props', () => {
+    const wrapper = mount(AcvPagination, {
+      props: {
+        total: 100,
+        limit: 10,
+        modelValue: 1,
+      } as AcvPaginationProps,
+    });
+
+    expect(wrapper.props()).toMatchInlineSnapshot(`
+      {
+        "limit": 10,
+        "modelValue": 1,
+        "total": 100,
+      }
+    `);
+  });
+
+  it('pass props', () => {
+    const wrapper = mount(AcvPagination, {
```

Comment:
this test case is almost the same as `default props`

Reply:
Maybe you wanted to test of updating props with **setProps**?

Reply:
ok, will add it

Reply:
not fixed, you can remove this case if its useless